src/frameworks/vue/template/server/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express');
+
+const app = express();
+const http = require('http').Server(app);
```

Comment:
Can we do all of our requires at the top then use those variables after? This will allow apps to upgrade to ES modules easier if they so desire.